lib/ansible/cli/pull.py

```diff
@@ -80,7 +80,7 @@ def init_parser(self):
 
         super(PullCLI, self).init_parser(
             usage='%prog -U <repository> [options] [<playbook.yml>]',
-            desc="pulls playbooks from a VCS repo and executes them for the local host")
+            desc="pulls playbooks from a VCS repo and executes them for the remote host")
```

Comment:
While I agree with #81130 that the current formulation is bad, I don't think changing 'local' to 'remote' makes it any better. I think this would be slightly better than the current version in `devel`:
```suggestion
            desc="pulls playbooks from a VCS repo and executes them locally")
```

Reply:
change to 'target' it is more agnostic and user can infer if local/remote depending on their context